File: day013/day013.cs
using System;
class Day013 {
  static void Main() {
    int x, y;
    
    Console.Write("Bir sayı giriniz: ");
    x = Convert.ToInt32(Console.ReadLine());
    
    Console.Write("Başka bir sayı giriniz: ");
    y = Convert.ToInt32(Console.ReadLine());
    
    if (x < y)
        Console.Write(y + " büyüktür " + x); 
    else if (x > y)
        Console.Write(x + " büyüktür " + y);        
    else
        Console.Write("Girdiğiniz sayılar eşittir. ");
  }
}

File: day013/day013.c
#include <stdio.h>

int main()
{
    int x, y;
    
    printf("Bir sayı giriniz: ");
    scanf("%d", &x);
    printf("Başka bir sayı giriniz: ");
    scanf("%d", &y);
    
    if (x < y)
        printf("%d büyüktür %d", y, x);
    
    else if (x > y)
        printf("%d büyüktür %d", x, y);
        
    else
        printf("Girdiğiniz sayılar eşittir.");

    return 0;
}
